src/elements/content-sidebar/SidebarNav.js

```diff
@@ -61,6 +61,7 @@ const SidebarNav = ({
                         data-testid="sidebaractivity"
                         sidebarView={SIDEBAR_VIEW_ACTIVITY}
                         tooltip={<FormattedMessage {...messages.sidebarActivityTitle} />}
+                        aria-label={intl.formatMessage(messages.sidebarActivityTitle)}
```

Comment:
We usually try to keep our attributes alphabetized, even though we have the lint rule disabled in this project.

Reply:
@jstoffan I will fix this. Thanks